src/rules/no-focused-tests.ts

```diff
@@ -71,7 +71,18 @@ export default createRule({
           callee.object.type === AST_NODE_TYPES.Identifier &&
           isCallToFocusedTestFunction(callee.object)
         ) {
-          context.report({ messageId: 'focusedTest', node: callee.object });
+          context.report({
+            messageId: 'focusedTest',
+            node: callee.object,
+            fix(fixer) {
```

Comment:
these should be suggestions as they might break user code

https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions

Reply:
yep - it's not a safe fix to automatically apply, but users can opt-in on a case by case basis

Reply:
Then in the README.md, should it be suggest or fixable?

Reply:
for focused test and people having autofix on save this would mess with their workflow as eslint would remove the focus when they save the change to run the tests

Reply:
>Then in the README.md, should it be suggest or fixable?

suggest 🙂